timescaledb/overview/release-notes/index.md

```diff
@@ -33,7 +33,7 @@ functionality works, and, as a result, made APIs simpler and more consistent,
 while also empowering users with more control and flexibility to customize
 behaviors to suit your needs.  Some of these API updates are **breaking changes**.
```

Comment:
```suggestion
behaviors to suit your needs.  
Some of these API updates are **breaking changes**.
```

Reply:
Does this reference to breaking change still refer to the 1.x to 2.0 upgrade or something else?

Perhaps if just the update to 2.0, we should delete this line at this point to avoid confusion, as I otherwise didn't know if it was suggesting breaking changes to 2.3/2.4.

Reply:
1.x to 2.0 only.